scst/src/scst_targ.c

```diff
@@ -3875,6 +3871,19 @@ static int scst_finish_cmd(struct scst_cmd *cmd)
 		}
 	}
 
+	if (likely(cmd->tgt_dev != NULL)) {
+		/*
+		 * We must decrement @tgt_dev->tgt_dev_cmd_count
+		 * after scst_tgt_cmd_done() was called.
```

Comment:
Please explain why in this comment.

Reply:
I have added problem description, but I'm not sure if a full explanation of the problem needs to be added.

I also have added the link to this PR, but I'm not sure if that's acceptable.